app/models/contact_details.rb

```diff
@@ -1,11 +1,11 @@
 class ContactDetails < ApplicationRecord
   MAX_LENGTHS = {
-    address: 250,
+    phone_number: 35,
     email_address: 250,
-    phone_number: 35
+    address: 250
```

Comment:
Hashes are unordered so I'm assuming you're doing this for readability, in which case 🌟 📐

Reply:
Ah so I actually changed the order of the hash as it's used for validating the maximum lengths of the fields in 

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/68efcdf31daeea40d67b47c8e4094fad1c96d281/app/models/contact_details.rb#L10

so the ordering of the hash is of significance.